src/components/Header.tsx

```diff
@@ -2,81 +2,133 @@ import React from 'react'
 import { AirDBContext } from '../context/AirDBContext'
 import {
   AppBar,
-  Toolbar,
   Typography,
-  Backdrop,
-  CircularProgress,
-  Fab,
+  Fade,
+  Grid,
+  IconButton,
+  Zoom,
   useTheme
 } from '@material-ui/core'
+import loader from './loader.gif'
 
-import { Close } from '@material-ui/icons'
-import Book from './icons/Book'
+import {
+  Add,
+  ExitToApp,
+  BubbleChart,
+  MoreHoriz,
+  Close
+} from '@material-ui/icons'
 
 const Header = (props: any) => {
-  const { loading } = React.useContext(AirDBContext)
+  const [toggled, setToggled] = React.useState(false)
+  const { loading, users, updateAirDB } = React.useContext(AirDBContext)
   const theme = useTheme()
+  const user = users.filter(
+    user => user.fields.username === window.localStorage.getItem('username')
+  )[0] || { fields: { active: 'false' } }
+  const loggedIn = !JSON.parse(user.fields.active)
+
+  const handleLogout = () => {
+    const userId = users.filter(
+      user => user.fields.username === window.localStorage.getItem('username')
+    )[0].id
+
+    updateAirDB('Testy - Users', userId, {
+      active: 'false'
+    }).then(() => {
+      props.setRedirect('/')
+      window.localStorage.removeItem('username')
+    })
+  }
 
   return (
     <>
       <AppBar
-        position="fixed"
         color="inherit"
         style={{
           display: 'flex',
           alignItems: 'left',
-          top: 'auto',
           background: theme.palette.background.paper,
-          bottom: 0,
           zIndex: 500
         }}
       >
         <div
           style={{
             width: '100%',
-            height: '2rem',
-            background: theme.palette.background.default,
-            borderBottom: '0.2em solid',
-            borderColor: theme.palette.primary.main
+            background: theme.palette.background.default
           }}
         />
-        <Toolbar variant="regular">
+        <div
+          style={{
+            width: '85%',
+            height: '5em',
```

Comment:
Again, you really should use `rem` here instead of `em`. Happy to have a call with you if you would benefit from me explaining how to set things up to be scalable!

Reply:
That would be great whenever you're available. When works best?

Reply:
That would be great whenever you're available. When works best?